ksql-engine/src/main/java/io/confluent/ksql/materialization/ks/KsMaterializedSessionTable.java

```diff
@@ -73,7 +72,7 @@
       while (it.hasNext()) {
         final KeyValue<Windowed<Struct>, GenericRow> next = it.next();
 
-        if (intersects(next.key.window().startTime(), lower, upper)) {
+        if (windowStart.contains(next.key.window().startTime())) {
```

Comment:
Are we preserving the old semantics here?

The old intersects method seems to have inclusive lower and upper bounds, but the Guava range method doesn't appear to at first glance (I haven't tested this): https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/Range.java#L434

Reply:
Agreed that the use of `Range` goes beyond just inclusive. However, I'll be leveraging this in a latter PR i.e. it's intentional.